### DISCUSSION TITLE
Anyone is able to use PS3 Controller with DsHidMini to play Elden Ring?

### DISCUSSION THREAD

**Discussion by zeroarst** [nefarius/DsHidMini#209] [Support]
I followed the instruction and Steam can see it and Windows 10 can see it as well.

But in Elden Ring only X Axis / Y Axis, Z Axis, X Rotation and Y rotation are working, other buttons do not work.

Did anyone have the same issue and know how to solve it?

![image](https://user-images.githubusercontent.com/1834837/156878809-6942a8c6-04c9-42a7-99aa-0e075abba8e8.png) 
![image](https://user-images.githubusercontent.com/1834837/156879125-83c60fb4-dfcd-4290-9532-7580b414ede0.png)
![image](https://user-images.githubusercontent.com/1834837/156879188-84cdd805-8f1e-42c8-adbd-43cb18180c6a.png)

Before I have this problem. I have another old PC, which uses the old driver Shibari, I need to run it to use the controller, and it works perfectly in Elden Ring. After I bought this PC, I decided to use the latest DshHidMini, but it just dose not work..

**Comment by nefarius** [comment]
If it worked using Shibari, use DS4Windows mode which emulates an x360 controller so it works exactly the same as Shibari under the hood.

Cheers

**Reply by zeroarst** [reply]
Wow cannot believe it actually works..............after struggled for 2 days almost gave up......thank you so much!

**Reply by nefarius** [reply]
You're quite welcome, game on!

**Comment by Jayy001** [comment]
Hi there, having the same problem- sadly DS4Windows  doesn't work either. Any ideas?

**Reply by Jayy001** [reply]
Using the default x-input

**Reply by nefarius** [reply]
Well, *why* does it not work? What means "not work"? Do you have the same settings and version in place like the OP? Etc. etc. nobody can offer you meaningful help with that little information.

**Reply by Jayy001** [reply]
> Well, _why_ does it not work? What means "not work"? Do you have the same settings and version in place like the OP? Etc. etc. nobody can offer you meaningful help with that little information.

Windows 11,
Latest version of dshidmini,

"having the same problem" - I am literally having the exact same problem as he said,  only X Axis / Y Axis, Z Axis, X Rotation and Y rotation are working, other buttons do not work.

Using a PS3 controller,  N1158

**Reply by Jayy001** [reply]
Using DS4Windows basically make my controller do nothing e.g. it doesn't give any input at all

**Reply by nefarius** [reply]
How did you verify that? With one specific game or in the DS4Windows app or in gamepad tester website or SCPUser.exe or....

**Reply by Kanuan** [reply]
DS4Windows does nothing on its own. You need the companion software "DS4Windows", and specifically the Ryochan7's DS4Windows (version 3.0.18 as for the day this was written)

For more info: [DS4Windows Mode User Guide](https://vigem.org/projects/DsHidMini/DS4-Mode-User-Guide/)

**Reply by Jayy001** [reply]
I verified using that exact game, as well as this - when using xinput if I hit the PS button, the xbox gamebar opens up but with DS4Windows it doesn't do anything

**Reply by Jayy001** [reply]
> DS4Windows does nothing on its own. You need the companion software "DS4Windows", and specifically the Ryochan7's DS4Windows (version 3.0.18 as for the day this was written)
> 
> For more info: [DS4Windows Mode User Guide](https://vigem.org/projects/DsHidMini/DS4-Mode-User-Guide/)

Ah, that's probably why then! I just thought you needed to switch, I should have looked at the docs more. Lemme try it now 👍🏻

**Reply by nefarius** [reply]
Please **read the documentation**, you literally need the tool "DS4Windows" **in addition** to changing the mode as stated by @Kanuan for real, people, is it so hard to spend 30 seconds on reading? We spent hours writing guides so you can ignore them...

**Reply by Jayy001** [reply]
Okay, I did read the documents when installing - No need to assume I haven't read any of it, but when I switched it today (Keep in mind I had it installed for a few months) I had presumed it would work without any further setups - like xinput did.

**Reply by Jayy001** [reply]
As well as this, you could also refer me to the documentation first before instructing me to do anything? Maybe that would help and actully let me know what I need to read....thanks for the help anyway

**Reply by nefarius** [reply]
[It is literally linked in the installation article you claimed you read...](https://vigem.org/projects/DsHidMini/HID-Device-Modes-Explained/#ds4windows)

**Reply by Jayy001** [reply]
> [It is literally linked in the installation article you claimed you read...](https://vigem.org/projects/DsHidMini/HID-Device-Modes-Explained/#ds4windows)

I said this in my message, I had installed it **a few months ago** so I had no need to read the DS4Window part as my xinput was fine.

**Reply by nefarius** [reply]
Visiting the same site simply again instead of asking on a discussion of someone else would've resulted in immediate success, but I digress, this leads to nowhere.

**Reply by Jayy001** [reply]
I 100% agree however this only puts me at a disadvantage, it doesn't affect you in anyway - you can choose to respond whenever you like so i'm not quite sure why you had a go at me? Anyway, thank you for the help - I'm about to test it now

**Reply by nefarius** [reply]
I still get a notification and need to read it to see if it's relevant to me. So it greatly affects me in a way that is utmost annoying; avoidable noise in my inbox - of which I have many - and avoidable as in repeating a topic covered by countless FAQs and articles we spent hours of our time setting it up conveniently searchable on one unified place. I am not having a go at you as an individual, heck, we don't know each other, it's two strangers on the internet. But this overall trend of people generating a post first instead of spending 30 seconds in an online search for the solution is what gets to me, I come from a generation where nobody was available to ask, you had a README or a site or a blog post and figured out the rest from those resources. This is an alarming, annoying trend, when you ask a question online, you literally demand attention, one way or the other, involving others. If it can be avoided, avoid it. Study the available resources first, then when those are exhausted, be my guest to come back to seek specific help. Not the other way around.

**Reply by Jayy001** [reply]
> This is an alarming, annoying trend, when you ask a question online, you literally demand attention, one way or the other, involving others.

Hey, I'm not demanding any attention - I understand it can be annoying to have that notification (which you can turn off) however, I was not demanding any attention - I asked a message, and left it at that. I didn't ask you guys to "hurry up", or "help me now", I was perfectly fine with waiting. I also immediately apoligised for not re-reading them again, however I believe that small rant was slighty unjustified. I also do not see anything wrong with asking for help! That's the whole point of these discussions! All you had to do was just refer me to the doucments (like the other person kindly did) and leave it at that.

**Reply by Jayy001** [reply]
After reading the docs, and installing DS4Windows - I can now open the xbox gamebar with the playstation button, however Elden Ring still doesn't pick up any inputs.

**Reply by nefarius** [reply]
> All you had to do was just refer me to the doucments (like the other person kindly did) and leave it at that.

Ah yes, all **I** had to do is get out of my way to replace the functionality of a search engine and invest my time in return to provide a link that could be discovered on own efforts in half a minute two clicks away from this very Discussion tab. This replay shows again that my points are not understood.

**Comment by Jayy001** [comment]
Update: I read the docs, didn't make any difference. I found a way to fix it on my own 🙂

**Reply by nefarius** [reply]
Gentleman's move to not share with your peers what the actual fix was...

![image](https://user-images.githubusercontent.com/286631/157729362-19583d60-9bbf-4809-a45c-8e0a1a4dd397.png)
